### PR TITLE
OneStar Drip

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
adds the OneStar uniforms that have been sitting in asset storage for like 2 years. currently only admin spawnable as i dont know how to make them spawn. 
image

## Why It's Good For The Game
New Drip Good. More OneStar Content Good

## Changelog
:cl:
add: New OneStar, Coat, Hat and Uniform. Sprites By Armolistsky
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
This is my first proper pull request so tell me if i fucked up somewhere

File: code/modules/clothing/head/armor.dm [hunk 1]
```diff
@@ -535,3 +535,18 @@
...
+	siemens_coefficient = 1
+	price_tag = 2000
+	armor = list(
+		melee = 15,
+		bullet = 40,
+		energy = 40,
+		bomb = 10,
+		bio = 5,
+		rad = 5
+	)
```
Comment (MLGTASTICa) [inline code/modules/clothing/head/armor.dm:552]: please leave 2 empty code lines here.

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -370,3 +370,12 @@ obj/item/clothing/under/genericr
...
 	)
+
+//Onestar
+//obj/item/clothing/under/onestar
+	name = "Onestar uniform"
+	desc = "A Comfortable red skilk uniform. it is emblazened with the onestar insignia on the collar. strangely it seems to fit you perfectly"
+	icon_state = "OS_Uniform"
+	item_state = "OS_Uniform"
+	style = STYLE_HIGH
+	price_tag = 500
```
Comment (MLGTASTICa) [inline code/modules/clothing/under/miscellaneous.dm:381]: 2 empty lines here too

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -476,6 +476,24 @@
 	siemens_coefficient = 0
 	price_tag = 600
 	//Used ablative gear armor values and technomancer helmet/voidsuit values.
+//onestar
+/obj/item/clothing/suit/storage/onestar/jacket
+	name = "OneStar officer jacket"
```
Comment (Firefox13) [inline code/modules/clothing/suits/armor.dm:481]:
```suggestion
	name = "One Star officer jacket"
```

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -476,6 +476,24 @@
...
+	desc = "A rare stylish red jacket worn by onestar officals. it seems to be extremly durable and is strangely warm to the touch"
```
Comment (Firefox13) [inline code/modules/clothing/suits/armor.dm:482]:
```suggestion
	desc = "A rare stylish red jacket worn by One Star offciers. It seems to be extremly durable and is strangely warm to the touch."
```

File: code/modules/clothing/head/armor.dm [hunk 5]
```diff
@@ -535,3 +535,18 @@
 	name = "gray tanker helmet"
 	icon_state = "tanker_helmet_gray"
 
+/obj/item/clothing/head/armor/helmet/onestar
+	name = "OneStar officer cap"
```
Comment (Firefox13) [inline code/modules/clothing/head/armor.dm:539]:
```suggestion
	name = "One Star officer cap"
```

File: code/modules/clothing/head/armor.dm [hunk 6]
```diff
@@ -535,3 +535,18 @@
...
+	desc = "A fancy red and blue cap sporting the Onestar insignia."
```
Comment (Firefox13) [inline code/modules/clothing/head/armor.dm:540]:
```suggestion
	desc = "A fancy red and blue cap sporting the One Star insignia."
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 7]
```diff
@@ -370,3 +370,12 @@ obj/item/clothing/under/genericr
 		bio = 100,
 		rad = 30
 	)
+
+//Onestar
+//obj/item/clothing/under/onestar
```
Comment (Firefox13) [inline code/modules/clothing/under/miscellaneous.dm:375]: Due to this line being commented the values below are assigned to the tuxedo above, so... Did you even test it?

File: code/modules/clothing/under/miscellaneous.dm [hunk 8]
```diff
@@ -370,3 +370,12 @@ obj/item/clothing/under/genericr
...
+	name = "Onestar uniform"
+	desc = "A Comfortable red skilk uniform. it is emblazened with the onestar insignia on the collar. strangely it seems to fit you perfectly"
```
Comment (Firefox13) [inline code/modules/clothing/under/miscellaneous.dm:377]:
```suggestion
	desc = "Comfortable-looking red silk uniform. It's emblazoned with the One Star insignia on the collar. Strangely, it seems to fit you perfectly"
```

File: code/modules/clothing/head/armor.dm [hunk 9]
```diff
@@ -535,3 +535,18 @@
...
+	icon_state = "OS_Hat"
```
Comment (Firefox13) [inline code/modules/clothing/head/armor.dm:541]: pls avoid using uppercases in icon and item states for the sake of consistency
if you think that `os_hat` or `os_jacket` would not be informative enough you can always change it to `onestar_hat` and `onestar_jacket`

File: code/modules/clothing/suits/armor.dm [hunk 10]
```diff
@@ -476,6 +476,24 @@
 	siemens_coefficient = 0
 	price_tag = 600
 	//Used ablative gear armor values and technomancer helmet/voidsuit values.
+//onestar
```
Comment (Firefox13) [inline code/modules/clothing/suits/armor.dm:479]: Could skip a line above for better readability, like you did in `code/modules/clothing/under/miscellaneous.dm`

**Review by Firefox13 — CHANGES_REQUESTED** [review]
Female under .dmi sprite name also needs renaming because it's genericr2-something and other ones need to be lowercased

File: code/modules/clothing/suits/armor.dm (hunk 4 above)
Comment (Firefox13) [reply code/modules/clothing/suits/armor.dm:482]: honestly that looks more like a coat to me

File: code/modules/clothing/suits/armor.dm (hunk 10 above)
Comment (ValoTheValo) [reply code/modules/clothing/suits/armor.dm:480]: Thanks for the suggestion.

**Comment by ValoTheValo** [comment]
welp i did all the changes people told me to do

File: code/modules/clothing/head/armor.dm (hunk 1 above)
Comment (ValoTheValo) [reply code/modules/clothing/head/armor.dm]: done

File: code/modules/clothing/under/miscellaneous.dm (hunk 2 above)
Comment (ValoTheValo) [reply code/modules/clothing/under/miscellaneous.dm]: and done

File: code/modules/clothing/suits/armor.dm (hunk 3 above)
Comment (ValoTheValo) [reply code/modules/clothing/suits/armor.dm]: Done

File: code/modules/clothing/suits/armor.dm (hunk 4 above)
Comment (ValoTheValo) [reply code/modules/clothing/suits/armor.dm]: Did the changes

File: code/modules/clothing/head/armor.dm (hunk 9 above)
Comment (ValoTheValo) [reply code/modules/clothing/head/armor.dm]: changed it

File: code/modules/clothing/under/miscellaneous.dm (hunk 7 above)
Comment (ValoTheValo) [reply code/modules/clothing/under/miscellaneous.dm]: fixed it and tested it. issue is resolved now

**Comment by ValoTheValo** [comment]
Life is pain

**Review by Firefox13 — APPROVED** [review]
Issues addressed

**Pull request closed without merging**